source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+Before we being, we're going to filter out low abundance/low prevelance ASVs. Filtering can provide better resolution and limit FDR penalty on features that are too far below the noise threshhold to be applicable to a statistical test. A feature that shows up with 10 counts may be a real feature that is present only in htat sample, may be a feature that's present in several samples but only got amplified and sequenced in one sample because PCR is a somewhat stocahastic process, or it may be noise. It's not possible to tell, so feature-based analysis may be better after filtering low abundance features. However, filtering also shifts the composition of a sample further disrupting the relationship. Here, the filtering is performed as trade off between the model, computation, and statistical 
+
+.. command-block::
+
+   qiime feature-table filter-features \
+     --i-table ./table_1k.qza \
+     --p-min-frequency 50 \
+     --p-min-samples 4 \
+     --o-filtered-table ./table_1k_abund.qza
+
+ANCOM operates on a ``FeatureTable[Composition]`` Artifact, which is based on the relative abundance of features on a per-sample basis. However, the ``FeatureTable[Composition]`` object cannot tolerate zeros (because compositional methods typically use a log-transform or a ratio and you can’t take the log or divide by zeros). To remove the zeros from our table, we add a pseudocount to the ``FeatureTable[Frequency]`` object.
+
+.. command-block::
+
+   qiime composition add-pseudocount \
+     --i-table ./table_1k_abund.qza \
+     --o-composition-table ./table1k_abund_comp.qza
+
+Let’s use ANCOM to check whether there is a difference in the mice based on their donor and then by their genetic background. The test will calculate the number of ratios between pairs of ASVs are significantly different with fdr-corrected p < 0.05.
+
+.. command-block::
+
+   qiime composition ancom \
+     --i-table ./table1k_abund_comp.qza \
+     --m-metadata-file ./metadata.tsv \
+     --m-metadata-column donor \
+     --o-visualization ./ancom_donor.qzv
+
+   qiime composition ancom \
+     --i-table ./table1k_abund_comp.qza \
+     --m-metadata-file ./metadata.tsv \
+     --m-metadata-column genotype \
+     --o-visualization ./ancom_genotype.qzv
+
+When you open the ancom visualizations, you’ll see a volcano plot on top which relates the ANCOM W statistical to the CLR (center log transform) for the groups. The W statistic is the number of tests whether the ratio between a given pair of ASVs is significant at the test threshold (typically fdr-adjusted p < 0.05). Because differential abundance in ANCOM is based on the ratio between tests, it does not produce a traditional p-value.
```

Comment:
```suggestion
When you open the ANCOM visualizations, you’ll see a volcano plot on top which relates the ANCOM W statistical to the CLR (center log transform) for the groups. The W statistic is the number of tests whether the ratio between a given pair of ASVs is significant at the test threshold (typically FDR-adjusted p < 0.05). Because differential abundance in ANCOM is based on the ratio between tests, it does not produce a traditional p-value.
```